openpype/tools/publisher/widgets/thumbnail_widget.py

```diff
@@ -438,6 +452,15 @@ def _on_clear_clicked(self):
         self.set_current_thumbnails()
         self.thumbnail_cleared.emit()
 
+    def _on_take_screenshot(self):
+        window = self.window()
+        window.setWindowState(QtCore.Qt.WindowMinimized)
+        output_path = os.path.join(
+            self._output_dir, uuid.uuid4().hex + ".png")
+        if capture_to_file(output_path):
+            self.thumbnail_created.emit(output_path)
+        window.setWindowState(QtCore.Qt.WindowActive)
```

Comment:
```suggestion
        window = self.window()
        states = window.windowStates()
        window.setWindowState(QtCore.Qt.WindowMinimized)
        output_path = os.path.join(
            self._output_dir, uuid.uuid4().hex + ".png")
        if capture_to_file(output_path):
            self.thumbnail_created.emit(output_path)
        # restore original window state
        window.setWindowStates(states)
```

Would it work to just restore the original states afterwards? Just to ensure if e.g. it was maximized or whatever it'd be again?

Another approach might be to not touch the window state at all and just set window visibility (which likely inheritly sets the states too?):

```python
        window = self.window()
        visibility = window.visibility()
        window.setVisibility(QtWidgets.QWindow.Hidden)
        output_path = os.path.join(
            self._output_dir, uuid.uuid4().hex + ".png")
        if capture_to_file(output_path):
            self.thumbnail_created.emit(output_path)
        # restore original window state
        window.setVisibility(visibility)
```